crates/biome_yaml_formatter/tests/specs/yaml/property/block_properties.yaml

```
@@ -0,0 +1,17 @@
+sequence: !!seq
+  - !!str a
+  - !!map
+    key: !!str value
+
+sequence:
```

YAMLlint (1.37.1)

[error] duplication of key "sequence" in mapping

(key-duplicates)